android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -378,7 +380,10 @@ class Screen(
         }
     }
 
-    private fun startTransitionRecursive(parent: ViewGroup?) {
+    private fun startTransitionRecursive(
+        parent: ViewGroup?,
+        isPossiblyRemovedClippedSubview: Boolean = true,
```

Comment:
Why is the default `true`? The only time we call it without this argument is at the start in `startRemovalTransition` yes? If so, maybe instead of giving the default, just use the proper value there too?

Reply:
+1

Reply:
@WoLewicki @kkafar you're absolutely right, it should be false by default or don't have a default value at all.
Eventually this mistake got me thinking and it looks like assuming it's true is a safe option here. Thus we can get rid of the redundant logic.